### PR TITLE
Feat/show page metadata

### PR DESCRIPTION
**Pull request by lucasbordeau**
- Working version of ObjectShowPage
- Fix useRecoilCallback race conditions
- Added pagination

### CODE REVIEW

File: front/src/modules/metadata/components/ObjectShowPage.tsx [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+      >
+        <PageFavoriteButton
+          isFavorite={false}
+          onClick={handleFavoriteButtonClick}
+        />
+        <ShowPageAddButton
+          key="add"
+          entity={{
+            id: object.id,
+            type: ActivityTargetableEntityType.Company,
```
Comment (Weiko) [inline front/src/modules/metadata/components/ObjectShowPage.tsx:98]: Is this intended to have Company here for now?

File: front/src/modules/metadata/utils/formatMetadataFieldAsColumnDefinition.ts [hunk 2]
```diff
@@ -35,4 +37,5 @@ export const formatMetadataFieldAsColumnDefinition = ({
   },
   Icon: IconBrandLinkedin,
   isVisible: true,
+  basePathToShowPage: `/object/${metadataObject.nameSingular}/`,
```
Comment (thaisguigon) [inline front/src/modules/metadata/utils/formatMetadataFieldAsColumnDefinition.ts:40]: Maybe we should use the metadataObject's slug?
```suggestion
  basePathToShowPage: `/object/${getObjectSlug(metadataObject)}/`,
```

File: front/src/modules/metadata/components/ObjectShowPage.tsx (hunk 1 above)
Comment (lucasbordeau) [reply front/src/modules/metadata/components/ObjectShowPage.tsx:98]: Yes, any other type isn't supported and we didn't support activities.

File: front/src/modules/metadata/utils/formatMetadataFieldAsColumnDefinition.ts (hunk 2 above)
Comment (lucasbordeau) [reply front/src/modules/metadata/utils/formatMetadataFieldAsColumnDefinition.ts:40]: Yes ! I'll implement it in another PR.

**Pull request merged**